service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/AbstractDocmosisOrder.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.groupingBy;
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+@Getter
+@AllArgsConstructor
+public class AbstractDocmosisOrder extends AbstractDocmosisData {
+    private final DocmosisJudgeAndLegalAdvisor judgeAndLegalAdvisor;
+    private final String courtName;
+    private final String familyManCaseNumber;
+    private final String dateOfIssue;
+    private final String complianceDeadline;
+    private final List<DocmosisRespondent> respondents;
+    private final List<DocmosisChild> children;
+    private final boolean respondentsProvided;
+    private final String applicantName;
+    private final DocmosisHearingBooking hearingBooking;
+    private final List<DocmosisDirection> directions;
```

Comment:
Again not every order has direction, this is abstract, but not really modelling the reality :(

Reply:
related to top comment